indico/web/forms/fields.py

```diff
@@ -476,6 +487,28 @@ def _value(self):
         return self.data or []
 
 
+class ReferencesField(MultipleItemsField):
+    """Extend `MultipleItemsField` to use a list of reference objects as the
```

Comment:
I think this class better fits into `indico/modules/events/fields.py` since it's nothing generic

For the dockstring, why not something like "A field to manage external references" - the fact that it inherits from MIF is not really relevant